common/persistence/persistence-tests/persistenceTestBase.go

```diff
@@ -364,7 +365,8 @@ func (s *TestBase) CreateWorkflowExecutionWithBranchToken(
 			Checksum:         testWorkflowChecksum,
 			VersionHistories: versionHistories,
 		},
-		RangeID: s.ShardInfo.RangeID,
+		RangeID:    s.ShardInfo.RangeID,
+		DomainName: s.DomainManager.GetName(),
```

Comment:
s.DomainManager.GetName() is not the domain name here.
BTW, since this is only in test, you can leave this empty.